datafusion/sqllogictest/test_files/ddl.slt

```diff
@@ -490,20 +490,28 @@ DROP TABLE csv_with_timestamps
 
 statement ok
 CREATE EXTERNAL TABLE csv_with_timestamps (
-  name VARCHAR,
-  ts TIMESTAMP,
   c_date DATE,
+  name VARCHAR,
```

Comment:
Are the changes to this test related / required? I think it would be better to leave this test as-is and add a new one if required so we can validate that we haven't inadvertently changed how CSV partitioned tables are read.

Reply:
I've just got a bit confusing result for JSON table and decided to also pin it in a test for CSV -- you're correct it's unrelated, and I'll better check for / file an issue regarding it. For now I've removed modifications for this file.

Reply:
Definitely feel free to file an issue if you came across an unrelated problem or something confusing where we could improve documentation.